src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs

```diff
@@ -65,6 +68,8 @@ internal NativeRequestContext(HttpApiTypes.HTTP_REQUEST* request, bool useLatin1
             _nativeRequest = request;
             _bufferAlignment = 0;
             _permanentlyPinned = true;
+
+            _backingBuffer = default!; // TODO: Won't
```

Comment:
What's up here?

Reply:
I think the comment was supposed to be "Won't ever be set when IIS uses this constructor".

Is this ok because `_permanentlyPinned` is set to true for IIS?

Reply:
Correct, IIS doesn't use the backing buffer.

Reply:
Ok. There are some methods on the context if called from IIS:

1. [`e19b529` (#28791)](https://github.com/dotnet/aspnetcore/pull/28791/commits/e19b52916f69159760ee7a780b8cc53510cd0f63#diff-3a0242fd0a04fb5ba4998e8765716d5fc15af5d8270f705c43c70e9e4ad86fd2R121)

2. [`e19b529` (#28791)](https://github.com/dotnet/aspnetcore/pull/28791/commits/e19b52916f69159760ee7a780b8cc53510cd0f63#diff-3a0242fd0a04fb5ba4998e8765716d5fc15af5d8270f705c43c70e9e4ad86fd2R553)

Reply:
Neither are called by IIS. Added comments that these will fail if called by IIS.